.github/workflows/ci.yml

```diff
@@ -18,24 +18,22 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v1
         with:
-          node-version: 14
-          
-      - name: Configure Artifactory credentials
-        run: |
-          # lel this is how jfrog wants you to come up with your scoped npm configuration
-          echo @momento:registry=https://momento.jfrog.io/artifactory/api/npm/npm-public/ >> ~/.npmrc
-          
+          node-version: '16.5.0'
+          registry-url: 'https://registry.npmjs.org'
+
       - name: Install Dependencies
-        run: npm ci
+        run: |
+          npm ci
+          node -v
 
       - name: Build
         run: npm run build
 
-      - name: Lint
```

Comment:
Why are we removing lint checks from ci.yml?

Reply:
I think Matt might turned it off since we're trying to figure out why the integration tests are failing.
This is a temporary measure. We'll put them back in.